src/dataDisplay/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,50 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap");
+
+.avatarWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  height: 40px;
+  width: 40px;
+}
+
+.templateAvatar {
+  border-radius: 50%;
+}
+
+.templateDefault {
+  position: relative;
+  border-radius: 50%;
+  background-color: #23A094;
+}
+
+.templateDefault p {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #FFFFFF;
+  margin: 0;
+}
+
+.avatarBig {
+  max-width: 40px;
+  width: 40px;
+  max-height: 40px;
+  height: 40px;
+}
+
+.avatarMedium {
+  max-width: 32px;
+  width: 30px;
+  max-height: 32px;
+  height: 32px;
+}
+
+.avatarSmall {
+  max-width: 24px;
```

Comment:
**nitpick:** Same as above

Reply:
hm... I had some troubles with sizes at some point and didn't check this after changing part of the HTML structure, now it's redundant